cloudformation/alarms.yaml

```diff
@@ -491,3 +491,93 @@ Resources:
       Period: 60
       Statistic: Maximum
       Threshold: 0
+  # cloudwatch synthetic support
+  MetadataCanary:
+    Type: AWS::Synthetics::Canary
+    Properties:
+      ArtifactS3Location: !Join ['', ['s3://', !Ref SyntheticsBucket, '/', !Join ['', [!Ref Stage, ig, canary]]]]
+      Code:
+        Handler: 'index.handler'
+        Script: |
+          var synthetics = require('Synthetics');
+          const log = require('SyntheticsLogger');
+
+          const apiCanaryBlueprint = async function () {
+    
+            // Handle validation for positive scenario
+            const validateSuccessful = async function(res) {
+              return new Promise((resolve, reject) => {
+                  if (res.statusCode < 200 || res.statusCode > 299) {
+                      throw res.statusCode + ' ' + res.statusMessage;
+                  }
+          
+                  let responseBody = '';

```

Comment:
Are these two blocks still needed if validation on responseBody is not performed?